### PR TITLE
media-sound/sequencer64: bump to 0.93.1

### LINKED ISSUE #74: New version 0.93.1 of sequencer64 available
A new version 0.93.1 of sequencer64 is available since 2017-08-11 19:58:17+00:00. For more details see https://github.com/ahlstromcj/sequencer64/releases

### PR DESCRIPTION
**Pull request by simonvanderveldt**
Fixes #74

### CODE REVIEW

**Comment by simonvanderveldt** [comment]
FYI for me 0.93.1 is unusable because of this bug https://github.com/ahlstromcj/sequencer64/issues/103
Might be a good idea if someone else can validate this behaviour, otherwise it might be better not to bump to 0.93.1

**Comment by mrVanDalo** [comment]
You can bump it and mask it.

File: media-sound/sequencer64/sequencer64-9999.ebuild [hunk 1]
```diff
@@ -29,10 +29,6 @@ RDEPEND="dev-cpp/gtkmm:2.4
 
 DEPEND="${RDEPEND}"
 
-PATCHES=(
```
Comment (mrVanDalo) [inline media-sound/sequencer64/sequencer64-9999.ebuild]: I would say this belongs to another commit, but I'm not to anal about it.
Do you know how to split commits ?

Comment (simonvanderveldt) [inline media-sound/sequencer64/sequencer64-9999.ebuild]: Sure, I'll split it into two commits

Comment (mrVanDalo) [inline media-sound/sequencer64/sequencer64-9999.ebuild]: sweet, I already approved. so you can merge right away

**Comment by simonvanderveldt** [comment]
I'll close this one for now, running into too many issues with this release (https://github.com/ahlstromcj/sequencer64/issues/103, https://github.com/ahlstromcj/sequencer64/issues/104, https://github.com/ahlstromcj/sequencer64/issues/106).

Don't think masking really makes sense when it doesn't work correctly at all.
